addons/jr/jr_classes.hpp

```diff
@@ -76,6 +77,20 @@ class asdg_OpticRail1913: asdg_OpticRail { // the "medium" rail, long enough to
         optic_KHS_hex = 1;
         optic_KHS_old = 1;
         optic_KHS_tan = 1;
+		//dpapexaddon

```

Comment:
> the 4 spaces for indent does not fit the files current structure as I see it

Yes it is. You can check the diff here to see the obvious difference.

> If spaces and indents are what you are worried about when a free lance community guys helps update then im helping the wrong people.

So you expect anyone can contribute with their own coding style? That would be an incredible mess. Every project needs to have some standard or you are asking for ugly code and a mess.